specification/columns/capacityreservationstatus.md

```diff
@@ -0,0 +1,43 @@
+# Capacity Reservation Status
+
+Capacity Reservation Status indicates whether the charge represents either the consumption of the [*capacity reservation*](#glossary:capacity-reservation) identified in the CapacityReservationId column or when the *capacity reservation* is unused.
+
+The CapacityReservationStatus column adheres to the following requirements:
+ * CapacityReservationStatus MUST be present in the billing data when the provider supports *capacity reservations* and MUST be of type String.
+ * CapacityReservationStatus MUST be null when CapacityReservationId is null.
+ * CapacityReservationStatus MUST NOT be null when CapacityReservationId is not null and [ChargeCategory](#chargecategory) is "Usage".
+ * CapacityReservationStatus MUST be one of the allowed values.
+ * CapacityReservationStatus MUST label all unused *capacity reservation* charges and MUST label used *capacity reservation* charges if the provider supports it.
```

Comment:
Is this saying that all rows need to have a CapacityReserationStatus? If so, that's already covered by the other requirements. If not, I'm confused by what this is saying. Given the requirements are already stated above and we don't have (or need) this for CommitmentDiscountStatus, I would propose we remove it.
```suggestion
```

Reply:
I think line 10 here just indicates labeling unused ODCR is always required, but labeling used ODCR is only required if supported by provider.